CONTRIBUTING.md

```diff
@@ -550,6 +550,21 @@ pull request. This PR is not signed and has the label `user-signing-required`. T
 done by a maintainer of the repository. Check out the PR and push an amending commit into
 the existing branch. Then, the PR can be merged.
 
+For an example, take a look at this pull request: [Bundle dependabot pull requests](https://github.com/weaveworks-liquidmetal/flintlock/pull/655).
+
+It was created by [this](https://github.com/weaveworks-liquidmetal/flintlock/actions/runs/4335326798) action triggered on the main branch.
+
+Once the pull request was opened, a maintainer created a commit into it by checking out the branch, then running a `git commit --amend`.
+
+```bash
+# git pull if not already up-to-date
+git checkout bundler-1678007734
```

Comment:
That does not work, only if you already have this branch created locally.

```
git checkout -b bundler-1678007734 origin/bundler-1678007734
```

Reply:
What do you mean. This totally works:

```
git branch -D bundler-1678007734
Deleted branch bundler-1678007734 (was 9e1faee).
➜  flintlock git:(main) git pull
Already up to date.
➜  flintlock git:(main) git checkout bundler-1678007734
branch 'bundler-1678007734' set up to track 'origin/bundler-1678007734'.
Switched to a new branch 'bundler-1678007734'
➜  flintlock git:(bundler-1678007734)
```

Reply:
It shouldn't 😆

Reply:
I'm not sure what it does when you have more than one `remote`.

Reply:
:D I'm not sure. So I have a test repo and created a branch on the UI then did a pull and I can checkout the branch. :D

```
git pull
remote: Enumerating objects: 10, done.
remote: Counting objects: 100% (8/8), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 10 (delta 5), reused 4 (delta 4), pack-reused 2
Unpacking objects: 100% (10/10), 3.09 KiB | 527.00 KiB/s, done.
From github.com:Skarlso/test
 * [new branch]      bundler-1677923771 -> origin/bundler-1677923771
 * [new branch]      bundler-1677929865 -> origin/bundler-1677929865
 * [new branch]      bundler-1677930003 -> origin/bundler-1677930003
 * [new branch]      bundler-1677930131 -> origin/bundler-1677930131
 * [new branch]      bundler-1677934413 -> origin/bundler-1677934413
 * [new branch]      bundler-1677935216 -> origin/bundler-1677935216
 * [new branch]      bundler-1677935458 -> origin/bundler-1677935458
 * [new branch]      bundler-1677935644 -> origin/bundler-1677935644
 * [new branch]      bundler-1677943918 -> origin/bundler-1677943918
 * [new branch]      test-checkout      -> origin/test-checkout
Already up to date.
➜  test git:(main) git checkout test-checkout
branch 'test-checkout' set up to track 'origin/test-checkout'.
Switched to a new branch 'test-checkout'
➜  test git:(test-checkout) git commit --amend
[test-checkout d29d2e2] Using the dependabot account
 Date: Sat Mar 4 16:30:58 2023 +0100
 1 file changed, 4 insertions(+), 2 deletions(-)
➜  test git:(test-checkout) git push -f
Enumerating objects: 9, done.
Counting objects: 100% (9/9), done.
Delta compression using up to 12 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (5/5), 1.15 KiB | 1.15 MiB/s, done.
Total 5 (delta 3), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
To github.com:Skarlso/test.git
 + d633b83...d29d2e2 test-checkout -> test-checkout (forced update)
➜  test git:(test-checkout)

```

But you're right that multiple remotes might mess with this. But then, I think it would be enough if you do a `git pull origin`. Anyways, I'll update to use your suggestion.

Reply:
Set it does not know which remote it should use unless you have `defaultRemote` in your config.
```
flintlock on  main via  v1.20.1
❯ git checkout bundler-1678007734
hint: If you meant to check out a remote tracking branch on, e.g. 'origin',
hint: you can do so by fully qualifying the name with the --track option:
hint:
hint:     git checkout --track origin/<name>
hint:
hint: If you'd like to always have checkouts of an ambiguous <name> prefer
hint: one remote, e.g. the 'origin' remote, consider setting
hint: checkout.defaultRemote=origin in your config.
fatal: 'bundler-1678007734' matched multiple (2) remote tracking branches

flintlock on  main via  v1.20.1
❯ git remote -v
origin  git@github.com:weaveworks-liquidmetal/flintlock.git (fetch)
origin  git@github.com:weaveworks-liquidmetal/flintlock.git (push)
second  git@github.com:weaveworks-liquidmetal/flintlock.git (fetch)
second  git@github.com:weaveworks-liquidmetal/flintlock.git (push)
```

Reply:
Updated.:)

Reply:
Yah, you're right.